index.html

```diff
@@ -301,6 +314,7 @@ <h2>Contact me</h2>
             </section>
         </footer>
     </main>
+    <script src="/script.js"></script>
```

Comment:
- [ ] after testing the application on my local machine I experience the same issues as the previous CR, however, I also found what the reason for this bug is. In this line you wrote `<script src="/script.js"></script>` although the correct format is `<script src="script.js"></script>`

Reply:
thank you so much for pointing out the right problem